forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/BiometricBindingHandler.kt

```diff
@@ -86,17 +89,14 @@ internal class BiometricBindingHandler(titleValue: String,
             when {
                 // API 29 and above, check the support for STRONG type
                 this.hasBiometricCapability(strongAuthenticators) -> {
-                    this.authenticatorType = AuthenticatorType.STRONG
```

Comment:
so these are the side effects?

Reply:
Yes, this is the side effect I mentioned, the method called "isSupported", it sounds like only checks if the device supports it, however, it changes state as well.